setup.py

```diff
@@ -101,6 +108,9 @@ def get_require_version(name):
         'Programming Language :: Python :: 3.4',
```

Comment:
IIRC trytond dropped support for this version of python and it will actually not work. tox.ini does in fact not cover python3.4. Maybe this marker should be dropped?